Makefile

```diff
@@ -41,19 +41,30 @@ $(PYTHON_DEPENDENCY_FILE): $(REQUIREMENTS_FILES)
 
 
 .PHONY: lint                                                                            ## Lints the code for all available runtimes
-lint:
-	cfn-lint -t $(SRC_FILES)
+lint: lint-templates lint-sam-apps
```

Comment:
The root Makefile is now for all project level automation. We could even do away with the sam app Makefiles, though I think it's convenient for developing.